source/codegen/metadata/nirfmxbluetooth/attributes.py

```diff
@@ -187,6 +187,17 @@
         'name': 'HIGH_DATA_THROUGHPUT_PACKET_FORMAT',
         'type': 'int32'
```

Comment:
Please update the README's [Supported NI Drivers](https://github.com/ni/grpc-device?tab=readme-ov-file#supported-ni-drivers) table for these RFmx personalities you're updating so it's up to date.

We copy that table in our Release notes to give a user an idea of how up to date each proto file is compared to the driver version.